pkg/controller/certificates/certificate_controller.go

```diff
@@ -79,29 +79,29 @@ func NewCertificateController(
 	csrInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			csr := obj.(*certificates.CertificateSigningRequest)
-			klog.V(4).Infof("Adding certificate request %s", csr.Name)
+			klog.V(4).InfoS("Adding Certificate request", "CSR", csr.Name)
 			cc.enqueueCertificateRequest(obj)
 		},
 		UpdateFunc: func(old, new interface{}) {
 			oldCSR := old.(*certificates.CertificateSigningRequest)
-			klog.V(4).Infof("Updating certificate request %s", oldCSR.Name)
+			klog.V(4).InfoS("Updating Certificate request", "oldCSR", oldCSR.Name)
```

Comment:
You seem to using a rule for field names of start with lower case unless its an acronym. While that makes a certain amount of sense I want to make sure we're adhering to our standards and it looks a tad odd.
/cc @serathius to check out standards on field names.

Reply:
@scottilee opened up an issue to track this: https://github.com/kubernetes/kubernetes/issues/100843

Reply:
@cheftako thanks for the comment. I'm not sure there are any standards, if there are could you point me to them? For instance sometimes I see [uppercased resources](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/certificates/certificate_controller.go#L100) and sometimes I'll see [lowercased resources](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/certificates/certificate_controller.go#L177) in the same file.

Reply:
Not sure there are any standards either. I tagged in those most likely to know, as I would like to find out.

Reply:
I'm removing the upper-casing I did.